web_src/css/shared/issuelist.css

```diff
@@ -3,6 +3,30 @@
   margin-top: 1rem;
 }
 
+.issue.list .item {
+  display: flex;
+  align-items: baseline;
+  padding: 8px 0;
+}
+
+.issue.list .item .issue-item-left {
+  display: flex;
+  align-items: start;
```

Comment:
Would `flex-start` also work? It has better browser support, and I don't understand the difference between them.

Reply:
Should work...

Reply:
Then use it to better support browsers like PaleMoon which may not have it yet.